### PR TITLE
Load CSS asynchronously + critical css

### PR DESCRIPTION
**Pull request by Tabrisrp**

### CODE REVIEW

File: inc/admin/options.php [hunk 1]
```diff
@@ -741,6 +746,26 @@ function rocket_settings_callback( $inputs ) {
 	}
 
 	/*
+	 * Option : critical CSS content for async CSS
```
Comment (GeekPress) [inline inc/admin/options.php]: If we don't use it anymore, we should remove it, nope?

Comment (Tabrisrp) [inline inc/admin/options.php]: We still need the field to input the critical css, I only removed the automatic generation of it

Comment (GeekPress) [inline inc/admin/options.php]: I'm talking about the commented PHP line in fact ^^

Comment (Tabrisrp) [inline inc/admin/options.php]: oops it should not be, will fix

**Pull request merged**